novuhq/website
# feat: oss friends page

This pull request brings "Our Open-source Friends" page

Steps to check:

- open preview [page](https://deploy-preview-94--novu-website.netlify.app/oss-friends/)
- make sure that everything looks and feels as expected
